src/status_im/contexts/communities/actions/accounts_selection/effects.cljs

```diff
@@ -87,6 +93,7 @@
       [:or [:set string?]
        [:sequential string?]]]
      [:airdrop-address string?]
+     [:share-future-addresses boolean?]
```

Comment:
We normally suffix booleans with a question mark (guideline here https://github.com/status-im/status-mobile/blob/24b77811d824faa2ad2741e617474d1b5e241200/doc/new-guidelines.md#use-a-question-mark-to-convey-the-value-is-a-boolean)

Reply:
yeah it would clear up a lot of what the variable is doing with this 👍